prov/verbs/src/verbs_msg_ep.c

```diff
@@ -532,16 +533,73 @@ fi_ibv_prepare_signal_send(struct fi_ibv_msg_ep *ep, struct ibv_send_wr *wr,
 	return FI_SUCCESS;
 }
 
+static inline int fi_ibv_poll_reap_unsig_cq(struct fi_ibv_msg_ep *ep)
+{
+	struct fi_ibv_wre *wre;
+	struct fi_ibv_wce *wce;
+	struct util_buf_pool *wre_pool;
+	struct ibv_wc wc[10];
+	int ret, i;
+
+	while (1) {
+		ret = ibv_poll_cq(ep->scq->cq, 1, wc);
```

Comment:
typo: it should be 10 for num_entries arg.

Reply:
Good catch! Done